securepy/restrictor.py

```diff
@@ -1,16 +1,24 @@
 import traceback
 import typing as t
 
+from securepy.stdio import IOCage
 from securepy.security import RESTRICTED_GLOBALS, SAFE_GLOBALS
-from securepy.stdcapture import StdCapture
 from securepy.timing import CapturingTimedFunction
 
 
 class Restrictor:
     """
     Prepare isolated python exec session
     """
-    def __init__(self, max_exec_time: int, max_std_memory: int = 100_000, restriction_scope: t.Literal[1, 2, 3] = 2):
+    def __init__(
+        self,
+        max_exec_time: int = 3,
+        max_std_memory: int = 100_000,
+        restriction_scope: t.Literal[1, 2, 3] = 2,
+        stdin=None,
```

Comment:
We should properly define the type here:
```suggestion
        stdin: t.Optional[str] = None,
```